returns/primitives/container.py

```diff
@@ -66,6 +66,12 @@ def __setstate__(self, state: Union[_PickleState, Any]) -> None:
             # backward compatibility with 0.19.0 and earlier
             object.__setattr__(self, '_inner_value', state)  # noqa: WPS609
 
+    def __replace__(self, **changes: Any) -> 'BaseContainer':
```

Comment:
Ok, I was wrong about adding this method to `BaseContainer` base class. I think that we need to add this to `Unwrappable` (?) instead. Basically, right now - this is not type-safe at all.

And add a proper type annotations. And add type tests for it.